src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -53,7 +53,7 @@ public class CustomFinder : IFindSagas<TestSaga08.SagaData08>.Using<SomeOtherMes
                         Id = Guid.NewGuid()
                     };
                     //Make sure saga exists in the store. Persisters expect it there when they save saga instance after processing a message.
-                    await SagaPersister.Save(sagaInstance, SagaCorrelationProperty.None, storageSession, new ContextBag()).ConfigureAwait(false);
+                    await SagaPersister.Save(sagaInstance, SagaCorrelationProperty.None, storageSession, (ContextBag)context).ConfigureAwait(false);
```

Comment:
that cast doesn't look like valid code to write. Can't we pass a new contextbag in here?